src/test/java/redis/clients/jedis/modules/RedisModuleCommandsTestBase.java

```diff
@@ -11,6 +11,7 @@
 import redis.clients.jedis.Protocol;
 import redis.clients.jedis.UnifiedJedis;
 import redis.clients.jedis.exceptions.JedisConnectionException;
+import redis.clients.jedis.json.parser.DefaultGsonParser;
```

Comment:
```suggestion
```

Reply:
Clear. I just forgot to format the code. Thanks.